CHANGES.md

```diff
@@ -1,4 +1,4 @@
-# 2.0.0 (UNRELEASED)
+# 2.0.0 (2018-02-12)
 - [NEW] Added API for upcoming IBM Cloud Identity and Access Management support
   for Cloudant on IBM Cloud. Note: IAM API key support is not yet enabled in the
   service.
```

Comment:
` - [NEW] Allow custom service name in CloudFoundry VCAP_SERVICES environment
variable.`

This sounds a bit weird to me because the change is in what nodejs-cloudant is able to extract from the environment variable not to the environment variable itself.

Maybe something like:
` - [NEW] Allow use of a custom service name from the CloudFoundry VCAP_SERVICES environment
variable.`

Reply:
I'd also like to see the `[IMPROVED]` entry moved from between the two `[REMOVED]` entries.